### PR TITLE
English Check #3

### PR DESCRIPTION
**Pull request by Crissman**
Add ability to do on CPU for PCs without NVIDIA GPU
Fix minor typos
Upgrade for Chainer v2
Remove error message

### CODE REVIEW

File: 3-Write-your-own-network_en.ipynb [hunk 1]
```diff
@@ -39,6 +39,8 @@
     "import chainer.functions as F\n",
     "import chainer.links as L\n",
     "\n",
+     "gpu_id = 0  # Set to -1 if you don't have a GPU\n",
```
Comment (mitmul) [inline 3-Write-your-own-network_en.ipynb]: I think it isn't needed to be placed here. I think it's better to put this line right before the line which performs model instantiation (`model = train(MyModel(10), gpu_id=gpu_id))`). How do you think about this?

File: 3-Write-your-own-network_en.ipynb [hunk 2]
```diff
@@ -134,7 +126,8 @@
     "\n",
     "    # 3. Model\n",
     "    model = L.Classifier(model_object)\n",
-    "    model.to_gpu(gpu_id)\n",
+    "    if gpu_id >=0:\n",
+    "            model.to_gpu(gpu_id)\n",
```
Comment (mitmul) [inline 3-Write-your-own-network_en.ipynb]: Please replace these 8 spaces with 4 spaces.

File: 3-Write-your-own-network_en.ipynb [hunk 3]
```diff
@@ -167,7 +160,7 @@
     "    \n",
     "    return model\n",
     "    \n",
-    "model = train(MyModel(10))"
+    "model = train(MyModel(10), gpu_id=gpu_id)"
```
Comment (mitmul) [inline 3-Write-your-own-network_en.ipynb]: Please insert `gpu_id = 0  # Set to -1 if you don't have a GPU` before this line.

File: 3-Write-your-own-network_en.ipynb [hunk 4]
```diff
@@ -415,7 +408,8 @@
     "        h = F.relu(self.bn(self.conv(x)))\n",
     "        if self.pool_drop:\n",
     "            h = F.max_pooling_2d(h, 2, 2)\n",
-    "            h = F.dropout(h, ratio=0.25, train=self.train)\n",
+    "            chainer.using_config('train', self.train)\n",
```
Comment (mitmul) [inline 3-Write-your-own-network_en.ipynb]: I think this line is not needed. `F.dropout` automatically switch the mode according to the global configuration, and it should be able to be the test mode when the global config of `train` is set to `False`. The global configuration is switched by `Trainer` and `Evaluator` automatically.

File: 3-Write-your-own-network_en.ipynb [hunk 5]
```diff
@@ -427,7 +421,8 @@
     "        self.train = True\n",
     "        \n",
     "    def __call__(self, x):\n",
-    "        return F.dropout(F.relu(self.fc(x)), ratio=0.5, train=self.train)"
+    "        chainer.using_config('train', self.train)\n",
```
Comment (mitmul) [inline 3-Write-your-own-network_en.ipynb]: This line also should be removed due to the same reason as the above.

File: 3-Write-your-own-network_en.ipynb (hunk 4 above)
Comment (mitmul) [reply 3-Write-your-own-network_en.ipynb]: And [L:404](https://github.com/mitmul/chainer-handson/pull/4/files#diff-c18291b98c19eec59e574bc5dc3e3166R404) should be removed for v2. Because the train/test mode is managed by `chainer.config.train`, and it is automatically switched by `Trainer` and `Evaluator`.

**Review by mitmul — CHANGES_REQUESTED** [review]
Thank you for the PR. I added some comments, so could you revise this PR?

And the part from L:466 to L:477 is not needed. Because switching the model from the training mode to the testing mode and vise versa will be done by Trainer and Evaluator implicitly. Could you remove this part?

**Comment by mitmul** [comment]
And the recommended way to add `Link` s to a `Chain` has been changed from V2. For example,

```
    def __init__(self, n_out):
        super(MyModel, self).__init__(
            conv1=L.Convolution2D(None, 32, 3, 3, 1),
            conv2=L.Convolution2D(32, 64, 3, 3, 1),
            conv3=L.Convolution2D(64, 128, 3, 3, 1),
            fc4=L.Linear(None, 1000),
            fc5=L.Linear(1000, n_out)
        )
```
the above part should be like below:

```
    def __init__(self, n_out):
        super(MyModel, self).__init__()
        with self.init_scope():
            self.conv1 = L.Convolution2D(None, 32, 3, 3, 1)
            self.conv2 = L.Convolution2D(32, 64, 3, 3, 1)
            self.conv3 = L.Convolution2D(64, 128, 3, 3, 1)
            self.fc4 = L.Linear(None, 1000)
            self.fc5 = L.Linear(1000, n_out)
```
Could you also fix `ConvBlock` and `LinearBlock` by following the same way to update this note for V2 completely?

**Comment by Crissman** [comment]
Updated as requested, tested on CPU and new code works. All `.train` lines removed.

**Comment by Crissman** [comment]
Additional chainer.Chain functions converted to v2 format
Japanese also revised to match English
Tested on GPU systems

Please merge!

**Review by mitmul — APPROVED** [review]
